### PR TITLE
User setting

### PR DESCRIPTION
**Pull request by Aurion195**

### CODE REVIEW

File: backend/app/tests.py [hunk 1]
```diff
@@ -9,20 +9,45 @@
...
-        
-        """
-                Avant chaque appel de fonction, on va faire cette fonction en 
-                premier
-        """
-        def setUp(self):
-                User.objects.create(username="Thomas", password="qwerty")
+        def testRegister(self):
+                """
+                Fonction permettant de tester le register et enregistrer une personne dans la
```
Comment (Slowblitz) [inline backend/app/tests.py:23]: In English pls

File: backend/app/tests.py [hunk 2]
```diff
@@ -9,20 +9,45 @@
...
+                        "firstname" : "Charly",
+                        "lastname" : "Croupier"}
+                
+                r = requests.post("Http://localhost:8000/register", data = data)
+                data = User.objects.get(username="legrosgarcia")
+                self.assertIsNotNone(data)
+                
+        def testLoginUser(self):
+                """
+                Fonction permmetant de tester la fonction login et son bon fonctionneent
```
Comment (Slowblitz) [inline backend/app/tests.py:38]: Same !

File: backend/app/tests.py [hunk 3]
```diff
@@ -9,20 +9,45 @@
...
+                """
+                data = {"username":"legrosgarcia", "mdp":"charly2020"}
+                r = requests.post("http://localhost:8000/login_user",  data = data)
+                expectResponseJson = {"connected":"True"}
+                self.assertEqual(expectResponseJson, r.json())
+
+        def testChangePassword(self):
+                """
+                Fonction permettant de tester le changement du mot de passe pour un utilisateur
```
Comment (Slowblitz) [inline backend/app/tests.py:47]: Same as above ! pls checkout all the docstrings

File: backend/app/views.py [hunk 4]
```diff
@@ -10,27 +10,92 @@
...
         Fonction permettant de récuper tous les utilisateurs enregistré
         dans la BDD
-"""
-def all_user(request):
+        
+        Args:
+                request ([type]): requête Http
+
+        Returns:
+                JsonResponse : la réponse en JSON de la fonction
```
Comment (Slowblitz) [inline backend/app/views.py:24]: Be more precise , give exemples

File: backend/app/views.py [hunk 5]
```diff
@@ -10,27 +10,92 @@
...
         y est il peut se connecter sinon il reste sur l'écran d'acceuil
-"""
-def login_user(request):
-        user = authenticate(username=request.POST["username"], password=request.POST["mdp"])
         
+        Args:
+                request ([type]): requête Http
+
+        Returns:
+                JsonResponse : la réponse en JSON de la fonction
```
Comment (Slowblitz) [inline backend/app/views.py:43]: Same =) .

**Review by Slowblitz — CHANGES_REQUESTED** [review]
You should not push *.pyc files pls add rules in .gitignore . Overall just change docstring and be more precise in  comments.

File: backend/app/tests.py (hunk 1 above)
Comment (Aurion195) [reply backend/app/tests.py:23]: fuck

**Review by Slowblitz — APPROVED** [review]

**Pull request merged**